docs/src/examples/components/Menu/Variations/MenuExampleToolbar.shorthand.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import { Menu, ToolbarBehavior, Icon, ToolbarButtonBehavior } from '@stardust-ui/react'
+
+const items = [
+  {
+    key: 'cloud',
+    icon: <Icon circular xSpacing="both" size="small" name="cloud" />,
```

Comment:
Not sure that we want to use JSX in shorthand examples?
You can define icon as JSON as well:
```js
    icon: {
      name: 'cloud',
      circular: true,
      xSpacing: 'both',
      size: 'small'
    }
```

Reply:
made the change thanks!